src/components/RegionVaccinationBlock/StateVaccinationBlock.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment } from 'react';
+import { Region } from 'common/regions';
+import Grid from '@material-ui/core/Grid';
+import { Heading2, Paragraph } from './RegionVaccinationBlock.style';
+import FeedbackBox from './FeedbackBox';
+import { StateLinkButton } from './StateVaccionationBlock.style';
+import { trackVaccinationLink } from './utils';
+
+const StateVaccinationBlock: React.FC<{ region: Region }> = ({ region }) => {
+  // TODO: Update with the eligibility and options links
+  const eligibilityUrl = '';
+  const optionsUrl = '';
+
+  if (!eligibilityUrl || !optionsUrl) {
```

Comment:
Not having the options url is potentially expected right?

Reply:
right, I meant `&&` 😅